whitepaper/deployment-entity-relationship-diagram.puml

```diff
@@ -0,0 +1,80 @@
+@startuml deployment-entity-relationship-diagram
+
+!include <material/common>
+' To import the sprite file you DON'T need to place a prefix!
+!include <material/cellphone>
+!include <material/laptop_chromebook>
+!include <material/database>
+
+title Deployment Entity Relationship Diagram\n\nHyperledger Cactus
+
+
+namespace Physical_Deplyoment_Elements {
+
+  entity CactusNode <<object>> {
+    * id: string
+    --
+    * nodeApiHost: string
+    * consortiumId: string
+    * memberId: string
+    * publicKeyPem: string
+    * plugins: Array<Plugin>
+    * ledgers: Array<Ledger>
+  }
+
+  entity APIServer <<OS process>> {
+    * plugins: Array<PluginInstance>
+  }
+
+  interface "ICactusPlugin" <<interface>> {
+    * getInstanceId(): string
+    --
+    * getPackageName(): string
+    * getPluginAspect(): string
+  }
+
+  entity "PluginInstance" <<object>> implements ICactusPlugin {
+    + instanceId: string
+    + pluginAspect: string
+    + packageName: string
+  }
+}
+
+entity "CactusConsortium" <<object>> {
```

Comment:
The word "Consortium" is misleading. I believe this is the registry of trusted Ledgers in CACTUS system. If this understanding is correct, I propose to change the name to "TrustedLedgerRegistry".

Reply:
> The word "Consortium" is misleading. I believe this is the registry of trusted Ledgers in CACTUS system.

@sfuji822 Apologies, I'm confused. Could you clarify a little further?
The `CactusConsortium` object has no property of it's own that is called/meant to to describe ledgers. 
It has a `members` property that holds an array of `ConsortiumMember` objects.

Reply:
> @sfuji822 Apologies, I'm confused. Could you clarify a little further?
> The `CactusConsortium` object has no property of it's own that is called/meant to to describe ledgers.
> It has a `members` property that holds an array of `ConsortiumMember` objects.

@petermetz  That sounds weird to me that multiple consortiums are hosted in a single place. IMHO, the set of Ledgers will be authorized (for system-wide) by consortium members when it is going to be part of the Integrated Service, and CACTUS application simply trusts the information (e.g. transactions) from Validators once it was added to a single registry(ConsortiumMember in your original wording). What do you think about my interpretation of a security model for integrated service?

Reply:
> That sounds weird to me that multiple consortiums are hosted in a single place.

@sfuji822 Let's tackle this first then because this seems to be where our mental models diverge. Maybe once we clarify this the rest will also get cleared up as well. So, `multiple consortiums are hosted in a single place` that sounds weird to me as well and I did not intend to give the impression of multiple consortiums being hosted in a single place either. Could you point me to the exact sentence/word that made you conclude that I am mentioning multiple consortiums and not just a single one? I want to make sure to somehow update that part to better express that I was not talking about multiple consortiums at all.